ydb/_topic_reader/topic_reader_asyncio.py

```diff
@@ -490,7 +490,11 @@ async def _start(self, stream: IGrpcWrapperAsyncIO, init_message: StreamReadMess
         logger.debug("reader stream %s send init request", self._id)
 
         stream.write(StreamReadMessage.FromClient(client_message=init_message))
-        init_response = await stream.receive()  # type: StreamReadMessage.FromServer
+        try:
+            init_response = await stream.receive(timeout=10)  # type: StreamReadMessage.FromServer
```

Comment:
1. What about use common named constant for init message for all stream requests, not literal digits.
2. Will the stream close after raise exception?